platform/ios/src/MGLAPIClient.m

```diff
@@ -106,7 +107,8 @@ - (void)setupUserAgent {
     NSString *appName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleName"];
     NSString *appVersion = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"];
     NSString *appBuildNumber = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"];
-    _userAgent = [NSString stringWithFormat:@"%@/%@/%@ %@", appName, appVersion, appBuildNumber, MGLAPIClientUserAgent];
+    NSString *sdkVersion = [NSBundle mgl_frameworkBundle].infoDictionary[@"CFBundleShortVersionString"];
```

Comment:
Come to think of it, what about using the `MGLSemanticVersionString` key instead, which will include, e.g., `-pre.1` for prereleases? Would distinguishing between final release and prerelease be useful?